lib/operation.js

```diff
@@ -57,11 +43,11 @@ const is = require('./is');
  * @throws {Error} Invalid parameters
  */
 function rotate (angle, options) {
-  if (this.options.useExifOrientation || this.options.angle || this.options.rotationAngle) {
+  if (this.options.angle || this.options.rotationAngle) {
```

Comment:
Dropping the check for auto-orientation here seems to be what the rotate function actually should do if it is going to warn. Calling `.rotate().rotate(90)` is a legit operation—even if we now prefer `.autoOrient().rotate(90)`. What we ought to warn about is an operation where commands are effectively dropped: `.rotate(90).rotate(180)`. I changed the test to reflect this as well.